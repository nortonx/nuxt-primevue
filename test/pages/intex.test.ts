import { shallowMount } from "@vue/test-utils";
import Index from "@/pages/index.vue";
import { describe, it, expect } from "vitest";

describe("Index page", () => {
  it("should render correctly and match snapshot", () => {
    const wrapper = shallowMount(Index);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
