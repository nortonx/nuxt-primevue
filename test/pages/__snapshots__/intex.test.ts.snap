// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Index page > should render correctly and match snapshot 1`] = `
"<div class="grid grid-nogutter surface-section text-800">
  <div class="lg:col-6 md:col-6 p-6 text-center md:text-left flex align-items-center">
    <section><span class="block text-6xl font-bold mb-1">Norton Almeida</span>
      <div class="text-6xl text-primary font-bold mb-3">Vue.js Frontend Developer</div>
      <p class="mt-0 mb-4 text-700 line-height-3">This is a landing page created using PrimeBlocks and PrimeVue, for demo purposes. Feel free to click on the links above and see the examples.</p>
      <button-stub label="Toggle Dialog Example" type="button" class="mr-3 p-button-raised"></button-stub>
      <button-stub label="Look, that's an AI generated image" type="button" icon="pi pi-arrow-right" class="p-button-outlined mt-2"></button-stub>
      <dialog-stub visible="false" modal="" header="Disclaimer" footer="- Norton Almeida" close-on-escape="" style="width: 50vw;"></dialog-stub>
    </section>
  </div>
  <div class="lg:col-6 md:col-6 overflow-hidden">
    <image-stub src="/images/AI_generated_2.jpeg" alt="Funko-like developer image" class="md:ml-auto block md:h-full" style="clip-path: polygon(8% 0, 100% 0%, 100% 100%, 0 100%);" preview=""></image-stub>
  </div>
</div>"
`;
